### PR TITLE
Add Octobox to Easypanel 🎉 

### PR DESCRIPTION
**Pull request by Supernova3339**

### CODE REVIEW

File: templates/octobox/index.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  const databasePassword = randomPassword();
+  const redisPassword = randomPassword();
+
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      env: [
+        `RAILS_ENV=production`,
```
Comment (Ponkhy) [inline templates/octobox/index.ts:19]: This either needs to be changed to `development` or the `SECRET_KEY_BASE` env needs to be added.

File: templates/octobox/index.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        `GITHUB_CLIENT_ID=${input.githubClient}`,
+        `GITHUB_CLIENT_SECRET=${input.githubSecret}`,
+        `OCTOBOX_DATABASE_NAME=${input.projectName}`,
+        `OCTOBOX_DATABASE_USERNAME=postgres`,
+        `OCTOBOX_DATABASE_PASSWORD=${databasePassword}`,
+        `OCTOBOX_DATABASE_HOST=${input.projectName}_${input.databaseServiceName}`,
+        `REDIS_URL=PAPERLESS_REDIS=redis://default:${redisPassword}@${input.projectName}_${input.redisServiceName}:6379`,
```
Comment (Ponkhy) [inline templates/octobox/index.ts:26]: Redis Server deployment is missing.

**Review by Ponkhy — CHANGES_REQUESTED** [review]
When it starts the Easypanel domain will be blocked. Maybe disable it or add a domain property and add it to the allow list?

**Comment by Supernova3339** [comment]
Whoops I'll fix that sorry

**Comment by Ponkhy** [comment]
The following env also need to be added `ALLOW_ALL_HOSTNAMES=true`.

**Comment by deiucanta** [comment]
@Supernova3339 do you want to fix this?

**Comment by Supernova3339** [comment]
> @Supernova3339 do you want to fix this?

Will be fixed ASAP

**Pull request merged**